src/syncData.ts

```diff
@@ -76,7 +85,9 @@ export const initSync = (
   const processChangeStreamRecords = async (docs: ChangeStreamDocument[]) => {
     try {
       const operations = []
+      const counts = getInitialCounts()
```

Comment:
can we also emit the `collection` for the counts? It should be really handy to log that so that we know what the counts apply to

Reply:
That is done on the consuming end since this library is only about syncing one collection.